src/scene/graphics/scene-grab.js

```diff
@@ -211,7 +211,7 @@ class SceneGrab {
                 if (camera.renderSceneColorMap) {
 
                     // allocate / resize existing RT as needed
-                    if (self.shouldReallocate(this.colorRenderTarget, camera.renderTarget?.colorBuffer, true)) {
+                    if (self.shouldReallocate(this.colorRenderTarget, camera.renderTarget, true)) {
```

Comment:
Also rename sourceTexture to sourceRT in the shouldReallocate function please.

Reply:
No problem. Done